src/styles/blocks/items.scss

```diff
@@ -0,0 +1,9 @@
+.items {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 942px;
```

Comment:
checklist 11th item, it should be an adaptive markup, use some type of formula wich will calculate container width based on cards-count and gap between them

Reply:
also that's what i see on lower resolution, it shouldn't look like that
<img width="731" alt="image" src="https://user-images.githubusercontent.com/37161315/157037007-3311dafc-538a-4738-b977-72b9676f89e0.png">